app/models/countryside_stewardship_grant.rb

```diff
@@ -0,0 +1,19 @@
+class CountrysideStewardshipGrant < Document
+
+  FORMAT_SPECIFIC_FIELDS = [
```

Comment:
You've used the `%i` notation elsewhere in this diff. We could take advantage of that here too, perhaps?